modules/raft/src/test/java/org/apache/ignite/raft/jraft/entity/codec/BaseLogEntryCodecFactoryTest.java

```diff
@@ -68,13 +69,11 @@ public void testEmptyData() {
 
     @Test
     public void testEncodeDecodeEmpty() {
-        try {
+        assertThrows(NullPointerException.class, () -> {
```

Comment:
Both `assertNull` `fail` are no longer need since `assertThrows` excepts an exception to be thrown.

> assertThrows(NullPointerException.class, () -> { this.encoder.encode(null); });

Reply:
you are right